src/borg/cache.py

```diff
@@ -531,7 +531,12 @@ def __init__(
             self.security_manager.assert_access_unknown(warn_if_unencrypted, manifest, self.key)
             self.create()
 
-        self.open()
+        try:
+            self.open()
+        except:
```

Comment:
Hmm, would it make sense to make this more specific?

A bare `except:` can often cause problems, especially if it is around a lot of code.

Reply:
Yes we could only catch FileNotFoundError and FileIntegrityError for now

Reply:
Yeah, that would be better!

If we find other exceptions fixable by the except block, we can add them later.